dapr/proto/runtime/v1/dapr.proto

```diff
@@ -541,6 +542,24 @@ message RegisteredComponents {
   repeated string capabilities = 4;
 }
 
+message PubsubSubscription {
+  string pubsub_name = 1;
+  string topic = 2;
+  map<string,string> metadata = 3;
+  PubsubSubscriptionRules rules = 4;
+  string dead_letter_topic = 5;
+  bool programmatic_subscription = 6;
```

Comment:
Why is this needed?

Reply:
the flag prgrammatic_subscription? I thought it would be useful to differentiate how the subscription was generated

Reply:
I would rather remove this to drive consistency across subscription methods.